DependencyInjection/Configuration.php

```diff
@@ -41,14 +41,32 @@ public function getConfigTreeBuilder() : TreeBuilder
 
         $rootNode
             ->children()
-                ->scalarNode('dir_name')->defaultValue('%kernel.root_dir%/DoctrineMigrations')->cannotBeEmpty()->end()
-                ->scalarNode('namespace')->defaultValue('Application\Migrations')->cannotBeEmpty()->end()
-                ->scalarNode('table_name')->defaultValue('migration_versions')->cannotBeEmpty()->end()
-                ->scalarNode('column_name')->defaultValue('version')->end()
-                ->scalarNode('column_length')->defaultValue(14)->end()
-                ->scalarNode('executed_at_column_name')->defaultValue('executed_at')->end()
-                ->scalarNode('all_or_nothing')->defaultValue(false)->end()
                 ->scalarNode('name')->defaultValue('Application Migrations')->end()
+                ->arrayNode('migrations_paths')
+                    ->requiresAtLeastOneElement()
+                    ->defaultValue([
+                        '%kernel.root_dir%/DoctrineMigrations' => 'Application\Migrations'
```

Comment:
As it's for 3.0, maybe we could move it to this and push the dependency for SF 4 and Flex and follow the existing Flex recipe:
```suggestion
                        '%kernel.project_dir%/src/Migrations' => 'App\Migrations'
```

Reply:
I personally agree. @alcaeus ?

Reply:
Yes.